freqtrade/freqtradebot.py

```diff
@@ -527,8 +527,6 @@ def execute_buy(self, pair: str, stake_amount: float, price: Optional[float] = N
             ticker_interval=timeframe_to_minutes(self.config['ticker_interval'])
         )
 
-        self._notify_buy(trade, order_type)
-
         # Update fees if order is closed
         if order_status == 'closed':
             self.update_trade_state(trade, order)
```

Comment:
I'm not sure here about these two lines -- if it's still okay to call notify_buy after update_trade_state?